.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,39 +1,48 @@
 ---
 name: Bug Report
 about: Template to report bugs
-title: ''
+title: 'bug: '
 labels: ''
 assignees: ''
 
 ---
 
-Description of the problem:
+# Bug Report
+
+## Capacitor Version
+<!-- Provide the version of Capacitor you are using in your application. This can be found in your package.json file. -->
+
+## Affected Platform(s)
 
-Affected platform
 - [ ] Android
 - [ ] iOS
-- [ ] electron
-- [ ] web
+- [ ] Electron
+- [ ] Web
+
+## Current Behavior
+<!-- Describe how the bug manifests. Be specific. -->
 
-OS of the development machine
-- [ ] Windows
-- [ ] macOS
-- [ ] linux
 
-Other information:
+## Expected Behavior
+<!-- Describe what the behavior would be without the bug. -->
 
-Capacitor version:
+## Sample Code or Sample Application Repo
 
-node version:
+<!-- If you are able to illustrate the bug or feature request with an example, please provide sample code snippets or a sample application via a public GitHub or Bitbucket repo. -->
```

Comment:
maybe add some note about issues with sample projects take priority as they are easier to reproduce